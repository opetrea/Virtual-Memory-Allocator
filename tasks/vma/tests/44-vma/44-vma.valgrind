==92270== Memcheck, a memory error detector
==92270== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
==92270== Using Valgrind-3.15.0 and LibVEX; rerun with -h for copyright info
==92270== Command: /home/student/Desktop/tema1_sda/sd-2023-homeworks/vma
==92270== Parent PID: 92269
==92270== 
==92270== 
==92270== HEAP SUMMARY:
==92270==     in use at exit: 0 bytes in 0 blocks
==92270==   total heap usage: 14 allocs, 14 frees, 8,504 bytes allocated
==92270== 
==92270== All heap blocks were freed -- no leaks are possible
==92270== 
==92270== For lists of detected and suppressed errors, rerun with: -s
==92270== ERROR SUMMARY: 0 errors from 0 contexts (suppressed: 0 from 0)
