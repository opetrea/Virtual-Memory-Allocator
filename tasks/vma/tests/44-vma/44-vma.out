Total memory: 0xC8 bytes
Free memory: 0xB4 bytes
Number of allocated blocks: 1
Number of allocated miniblocks: 2

Block 1 begin
Zone: 0xA - 0x1E
Miniblock 1:		0xA		-		0x14		| RW-
Miniblock 2:		0x14		-		0x1E		| ---
Block 1 end
